### ISSUE TITLE
Replacement for CLK_TCK on Windows build

### ISSUE THREAD

**Issue by ryanbogan** [opensearch-project/performance-analyzer#121]
I am currently writing an install script for a Windows build for issue [#828](https://github.com/opensearch-project/opensearch-build/issues/828). What should the replacement be for this section of `opensearch-tar-install.sh` on a Windows build?

```
CLK_TCK=`/usr/bin/getconf CLK_TCK`
echo "-Dclk.tck=$CLK_TCK" >> $OPENSEARCH_HOME/config/jvm.options
```

**Comment by dblock** [comment]
@ryanbogan Note that PA is [currently not included in the Windows distribution](https://github.com/opensearch-project/opensearch-build/blob/main/manifests/1.3.0/opensearch-1.3.0.yml#L59), so it's safe to skip this variable altogether for Windows right now.

**Comment by CEHENKLE** [comment]
@ryanbogan hey can we get an update on this?

**Comment by dbwiddis** [comment]
TLDR:  Set any value you want, Windows doesn't use this constant, but 10,000,000 is probably the best option.

Short answer:  If you have to pick a value, choose 10,000,000, which represents the number of increments of the displayed "ticks" value per second, which correlates directly to the practical usage (converting "jiffies" to a unit of time) of Linux CLK_TCK (ticks per second) and matches the [`DateTime.Ticks`](https://learn.microsoft.com/en-us/dotnet/api/system.datetime.ticks?view=net-7.0) constant in .NET.

Long answer:

Arguably, you could choose 64, which is the actual number of clock ticks per second, although it's never used in any calculation and while accurate, is useless. The windows clock changes in 1/64 of a second increments, which equates to 15.625 milliseconds.  (This is roughly comparable to the 10 millisecond ticks of most Linux distributions with CLK_TCK of 100.)   However, that value is never actually used in userspace.

However, the unit displayed to users for the System clock and any time-related metrics is in 100 nanosecond units.   (This is amusing as it appears to give an artificial precision, but the values increment in multiple of 156250.).
